docs/user_guide/HOWTO/IPA_Clients.rst

```diff
@@ -74,29 +77,43 @@ To be able to add hosts from the command line:
       # In <domain>.yaml or whichever level is most specific
       ---
       classes:
-      - simp::ipa::install
-      - simp::ipa::client
+      - simp_ipa::client::install
 
-      simp::ipa::install::ensure: present
-      simp::ipa::install::password: "%{trusted.certname}-<password from script above>"
-      simp::ipa::install::server: ipa.example.domain
-      simp::ipa::install::domain: example.domain
-      simp::ipa::install::realm: EXAMPLE.DOMAIN
+      simp_ipa::client::install::ensure: present
+      simp_ipa::client::install::password: "%{trusted.certname}-<password from script above>"
+      simp_ipa::client::install::server: ipa.example.domain
+      simp_ipa::client::install::domain: example.domain
+      simp_ipa::client::install::realm: EXAMPLE.DOMAIN
 
+      # This setting should already exist, it needs to be modified
+      sssd::domains:
+        - LOCAL
+        - <IPA Domain>
 
    Some optional settings may be needed, depending on the configuration of the
    IPA server and the environment:
 
    .. code-block:: yaml
 
       ---
+      # IPA's default uid's are in the millions while SIMP's max is much lower
+      simp_options::uid::max: 0
+
+      # IPA uses both of these technologies, so they need to be enabled
```

Comment:
comment is not clear.  I think you mean to say that if you are not using 'simp' or 'simp-lite' scenario, be sure to enable these features.